public/templates/finesse_match.html

```diff
@@ -45,9 +45,27 @@
       href="/assets/css/owl-carousel.css"
     />
 
-    <!-- import the webpage's javascript file -->
-    <script src="https://code.jquery.com/jquery-3.3.1.js"></script>
-    <script src="/components/patientSignUp.js" defer></script>
+    <!-- Scripts -->
```

Comment:
Just a heads up. This file will be deleted once my pr gets merged into the main repo. This file name gets changed to "demo.html". I'd say wait till my pr gets merged and then do a fetch/merge/rebase, and add these changes to that file.

Reply:
Let's move forward with this change. Steve, you'll have to resolve the merge conflict on your PR.